src/server/messages.ts

```diff
@@ -146,6 +151,9 @@ export function validate(filename: string, project: string, text: string) {
     console.log('Sent message');
 
     console.log(fullMessage.toString());
+
+    //TODO this is just a placeholder
+    return ServerResult(oracc_log);
```

Comment:
```suggestion
    return new ServerResult(oracc_log);
```

I think this is what caused the tests to fail?

Reply:
One of the things, yep. I did before seeing your suggestion, so not committing it.